app/recipe/serializers.py

```diff
@@ -0,0 +1,63 @@
+from rest_framework import serializers
+
+from core.models import Recipe, Ingredient
+
+
+class IngredientSerializer(serializers.ModelSerializer):
+    """
+    Serializer for ingredients
+    """
+
+    class Meta:
+        model = Ingredient
+        fields = ('name',)
+        read_only_fields = ('id',)
+
+
+def add_ingredients(recipe, ingredients):
```

Comment:
This should probably go inside of the `Recipe` model as one of its methods.  Thus you'll have better encapsulation.

Reply:
Also, another approach would be using a [model manager](https://docs.djangoproject.com/en/3.0/topics/db/managers/) where you could create a recipe with its ingredients this way: 
```
Recipe.objects.create_with_ingredients(recipe_data, ingredient_list)
```

Reply:
Changed as suggested. Implemented a new RecipeManager in order to abstract the ingredients creation. Now the Recipe creation method in RecipeManager accept Recipe name, description and ingredients.